test/intl402/Locale/prop-desc.js

```diff
@@ -2,21 +2,25 @@
 // This code is governed by the BSD license found in the LICENSE file.
 
 /*---
-esid: pending
+esid: sec-intl.locale
 description: >
-    Checks the properties of the Locale constructor.
+    The value of the [[Prototype]] internal slot of the Intl.Locale constructor is the
+    intrinsic object %FunctionPrototype%.
 info: |
-    Every other data property described in clauses 18 through 26 and in Annex B.2 has the attributes { [[Writable]]: true, [[Enumerable]]: false, [[Configurable]]: true } unless otherwise specified.
-    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2019 Language Specification, 10th edition, clause 17, or successor.
-    The Locale constructor is a standard built-in property of the Intl object.
+  The value of Intl.Locale.prototype is %LocalePrototype%.
+
+  This property has the attributes
+  { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: false }.
+
+includes: [propertyHelper.js]
 features: [Intl.Locale]
 ---*/
 
-const propdesc = Object.getOwnPropertyDescriptor(Intl, "Locale");
-assert.sameValue(propdesc.writable, true);
-assert.sameValue(propdesc.enumerable, false);
-assert.sameValue(propdesc.configurable, true);
-assert.sameValue(propdesc.value, Intl.Locale);
+assert.sameValue(typeof Intl.Locale, "function", "typeof Intl.Locale is function");
 
-assert.sameValue(typeof Intl.Locale, "function");
-assert.sameValue(Object.getPrototypeOf(Intl.Locale), Function.prototype);
+verifyProperty(Intl, "Locale", {
+  value: Intl.Locale,
+  writable: false,
+  enumerable: false,
+  configurable: false,
```

Comment:
This change seems to be all kinds of wrong. The description is no longer about what's being tested, and the expectations match the description, not the test.

Reply:
This test file ended up with a mismatch in description/info for the test content, as I was trying to break out the two things being asserted (per Test262 convention).